Sources/CMakeLists.txt

```diff
@@ -12,6 +12,31 @@
 ##
 ##===----------------------------------------------------------------------===##
 
+if(NOT SwiftFoudation_MACRO)
+  include(ExternalProject)
```

Comment:
The rest of our CMake files use 4-space indents, could we update to do the same here?

Reply:
Fixed